tests/Avalonia.IntegrationTests.Appium/ComboBoxTests.cs

```diff
@@ -28,7 +28,7 @@ public void Can_Change_Selection_Using_Mouse()
             comboBox.Click();
             _session.FindElementByName("Item 1").SendClick();
 
-            Assert.Equal("Item 1", comboBox.GetComboBoxValue());
+            Assert.Equal("Item 2", comboBox.GetComboBoxValue());
```

Comment:
Forcing a test failure here. The commit that makes this change needs to be removed before merge.